EIPS/eip-2390.md

```diff
@@ -0,0 +1,299 @@
+specific to the geographic location of the query.
+Unfortunately the ENS specification does not provide a mechanism for
+geo-specific resolution.
+ENS can respond to queries with IP addresses (as described in EIP1185)
+however there is no way to respond to geo-specific queries.
+This EIP proposes a standard to give the ENS system geo-proximal awareness
+to serve a similar purpose as GeoDNS.
+
+GeoENS can do more than DNS-based solutions.
+In addition to geographic split horizon DNS, GeoENS can be used for the following:
+ - Locating digital resources (like smart contracts) that represent physical objects in the real world.
+ - Smart contract managing access to a physical object associated with a specific location.
+ - ENS + IPFS web hosting (as described in EIP1162) with content translated to the native language of the query source.
+ - Tokenizing objects with a physical location.
+
+Because of the decentralized nature of ENS, geo-specific resolution is different than traditional GeoDNS.
+GeoDNS works as follows. DNS queries are identified by their source IP address.
+This IP is looked up in a database like [GeoIP2](https://www.maxmind.com/en/geoip2-services-and-databases)
+from MaxMind which maps the IP address to a location.
+This method of locating the source of a query is error prone and unreliable.
+If the GeoIP database is out of date, queried locations can be vastly different than their true location.
+GeoENS does not rely on a database because the user includes a location in their query.
+
+It follows that queries can be made by users for any location, not just their location.
+Traditional DNS will only return the resource assigned to a query's provenance.
+GeoENS does not correlate a query's provinance with a location, allowing the
+entire globe to be queried from a single location.
+
+An additional shortcoming of traditional DNS is the fact that there is no way to return a list of servers in a certain proximity.
+This is paramount for uses cases that require discovering the resource with the lowest latency.
+GeoENS allows a list of resources, like IP addresses, to be gathered within a specific location.
+Then a client to determine themselves which resource has the lowest latency.
+
+Lastly, publicly facing GeoDNS services do not give fine granularity control
+over geographic regions for GeoDNS queries.
+Cloud based DNS services like [Amazon's Route 53](https://aws.amazon.com/route53/)
+only allow specifying geographic regions at the granularity of a State in
+the United States.
+GeoENS on the other hand gives 8 characters of geohash resolution which
+corresponds to +-20 meter accuracy.
+
+## Specification
+This EIP proposes a new interface to ENS resolvers such that geo-spacial information
+can be recorded and retrieved from the blockchain.
+The interface changes are described below for "address resolvers" described in EIP137
+however the idea applies to any record described in EIP1185 and EIP1062, namely DNS
```

Comment:
All of these linked EIPs must link to the EIP text and should be formatted as EIP-137, ie. `[EIP-137](./eip-137.md)`

Reply:
Fixed in f5b3e8da0a8134f61c4df6fc0bd61bba12157823